docs/workflows/sdk-setup.md

```diff
@@ -0,0 +1,291 @@
+---
+title: SDK Setup
+sidebar_position: 8
+# Display h2 to h2 headings
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Overview
+
+The Embedded SDK is a holistic SDK solution offering the entire experience embedded in your product. Users will not need to download the Beyond Identity Authenticator. A set of functions are provided to you through the `Embedded` namespace. This SDK supports OIDC and OAuth2.
+
+## Sample App
+
+<Tabs groupId="sdks">
+<TabItem value="android" label="Android">
+
+:::tip Sample App
+Sample apps are available to explore. Check out [Example](https://github.com/gobeyondidentity/bi-sdk-android/tree/main/app) for the Embedded SDK.
```

Comment:
I see what you're doing here. But I was initially confused because the tip wasn't changing. I wonder if just for this section, we display them as boxes? Kind of how we do it in Console: 
<img width="1225" alt="image" src="https://user-images.githubusercontent.com/6578679/212430772-aa54db45-5746-451e-9268-0a0937da3196.png">

What do you think?

Reply:
I like it. Do you mean for the tabs? I'm not sure if images are supported, but I'm happy to play around

Reply:
Not for the tabs. Just as a replacement for the tip in the first section.

Reply:
I like this idea too. I was also a little confused when I first tapped the top tabs and the example text didn't change.

Reply:
I'm having some difficulty recreating these, but for now I have added the respective logos in the Sample App title

Reply:
This seems like a good compromise to me, though the JS logo looks a little grainy.

Reply:
I figured it out. Docusaurus prefers svgs as imports.